packages/badge/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@react-spectre/badge",
+  "version": "1.0.0-12",
+  "description": "React components for Spectre.css's badge elements.",
+  "jsnext:main": "src/index.js",
+  "main": "dist/badge.js",
+  "browser": "dist/badge.umd.js",
+  "module": "dist/badge.es.js",
+  "repository": "https://github.com/react-spectre/react-spectre/tree/master/packages/form",
+  "author": "Kenji Mukai <kensuka21@gmail.com>",
```

Comment:
I may replace this to avoid issues with NPM organizations and scoped packages. However, your credits will stay.